WalletWasabi/Nito/.editorconfig

```diff
@@ -2,4 +2,7 @@
 [*]
 
 # Workaround to suppress all warnings in the Nito folder
-generated_code = true
\ No newline at end of file
+generated_code = true
+
+# Suppress remaining nullable warnings
+dotnet_diagnostic.cs8669.severity = none
```

Comment:
I would also try to remove these lines and have

```
root = true

[*]
```

it might be the best solution as it means: "Do not take into account any .editorconfig files in parent folders (sounds like a good idea) and as there are no rules defined, no rules should be really applied. Hopefully.

In any case, specifying `root = true` seems beneficial (https://editorconfig.org/).

Reply:
Sadly this didn't solve it, the resolved errors reappear with this.